packages/cli/src/cmds/validator/keys.ts

```diff
@@ -88,8 +88,8 @@ function resolveKeystorePaths(fileOrDirPath: string): string[] {
   if (fs.lstatSync(fileOrDirPath).isDirectory()) {
     return fs
       .readdirSync(fileOrDirPath)
-      .map((file) => path.join(fileOrDirPath, file))
-      .filter((filepath) => filepath.endsWith(".json"));
+      .filter((file) => file.indexOf("deposit_data") === -1 && file.endsWith(".json"))
```

Comment:
A Regex expression is typically much faster than indexOf and endsWith and much more idiomatic

Reply:
> A Regex expression is typically much faster than indexOf  and endsWith
I think that is debatable. The benchmarks I see (not ones I run myself) suggests this is not the case.
> and much more idiomatic
Using regex is definitely more succinct, and since this is not a hot code path, I guess it is fine to go with the regex. 

I have updated.

Reply:
Good point I haven't benchmarked it :laughing: